osgi/itests/src/test/java/org/apache/cxf/osgi/itests/soap/JmsServiceTest.java

```diff
@@ -78,8 +78,10 @@ private static ActiveMQConnectionFactory createConnectionFactory() {
         return OptionUtils.combine(
             cxfBaseConfig(),
             features(cxfUrl, "cxf-jaxws", "cxf-transports-jms"),
+            features(maven().groupId("org.apache.karaf.features").artifactId("spring-legacy").versionAsInProject()
```

Comment:
spring-legacy feature descriptor added to avoid activemq-client feature installation error

Reply:
Could you add in a link to the AMQ issue here so that we know to remove it when 5.16.1 is released? Same for the spring-legacy dependency in the pom.

Reply:
added